modules/adopter-registration-impl/src/main/java/org/opencastproject/adopter/statistic/ScheduledDataCollector.java

```diff
@@ -408,4 +442,29 @@ public void setOrganizationDirectoryService(OrganizationDirectoryService orgDirS
     this.organizationDirectoryService = orgDirServ;
   }
 
+  @Reference
+  public void setTrustedHttpClient(TrustedHttpClient trustedHttpClient) {
+    this.httpClient = trustedHttpClient;
+  }
+
+  private class TobiraRemoteRequester extends RemoteBase {
+    private static final String SERVICE_TYPE = "org.opencastproject.tobira";
```

Comment:
Magic constant is magical, but I didn't think adding a compile time dependency was worth it here?